### DISCUSSION TITLE
Can't log in.   What more could they possibly want?

### DISCUSSION THREAD

**Discussion by Reasoning-Technology** [community/community#133182] [New to GitHub]
So you think you want to share some code?   ...

I haven't been able to get into github for several months because they said they required a phone number, then rejected the number we provided.  It turns out that they don't want a phone number, rather they want an individual's IMEI number.   Github is now only available to those who pay telecoms for regular cell phone service and by virtue of that have known banking credentials.

So OK,  so they now have my IMEI number registered with a major telecom provider, tied to a bank account.  I then added a hardware key.  As 2FA goes it does not get better than the hardware key.

Ok, they have IMEI number now, they have the security app, *and* we have 2FA though a hardware security key.  BUT THIS IS NOT  ENOUGH.   Today I log in and it says:
 
     Please update your two-factor authentication methods on this page, otherwise access will be limited. 

They also want me to update the hardware key to a pass key.
     
I have an easier time entering the NSA building than maintaining a fn github account for the startup so as to give away some code.  

What more can they possibly want?  What is this about?  That I hire a full time security consultant?  Give them biometrics?  DNA?  Do they want to hold a family member as collateral?

Github, this is over the top.  Please leave me in peace and let me get back to work on code.

**Comment by ghostinhershell** [comment]
Hi @Reasoning-Technology, 

Thanks so much for reaching out in our community. GitHub doesn't require that you provide an IMEI number in order to register for two-factor authentication. Can you clarify the issue that you're running into here?

We know 2FA may be a new process for some folks and with that comes questions. We put together a [FAQ](https://github.com/orgs/community/discussions/68240) that will help answer your concerns above. If you still have more questions, feel free to ask them here.
